Tests/FunctionalTestBundle/Resources/config/config.yml

```diff
@@ -3,12 +3,81 @@ parameters:
   fm_bbcode.hook.bar.class: Decoda\Hook\ClickableHook
 
 services:
+  fm_bbcode.decoda.filter.block:
+    class: '%fm_bbcode.decoda.filter.block.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: block }
+
+  fm_bbcode.decoda.filter.code:
+    class: '%fm_bbcode.decoda.filter.code.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: code }
+
+  fm_bbcode.decoda.filter.email:
+    class: '%fm_bbcode.decoda.filter.email.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: email }
+
+  fm_bbcode.decoda.filter.image:
+    class: '%fm_bbcode.decoda.filter.image.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: image }
+
+  fm_bbcode.decoda.filter.list:
+    class: '%fm_bbcode.decoda.filter.list.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: list }
+
+  fm_bbcode.decoda.filter.quote:
+    class: '%fm_bbcode.decoda.filter.quote.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: quote }
+
+  fm_bbcode.decoda.filter.text:
+    class: '%fm_bbcode.decoda.filter.text.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: text }
+
+  fm_bbcode.decoda.filter.url:
+    class: '%fm_bbcode.decoda.filter.url.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: url }
+
+  fm_bbcode.decoda.filter.default:
+    class: '%fm_bbcode.decoda.filter.default.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: default }
+
+  fm_bbcode.decoda.filter.video:
+    class: '%fm_bbcode.decoda.filter.video.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: video }
+
+  fm_bbcode.decoda.filter.table:
+    class: '%fm_bbcode.decoda.filter.table.class%'
+    public: true
+    tags:
+        - { name: fm_bbcode.decoda.filter, id: table }
+
```

Comment:
This duplicate service definition on `Resources/config/filters.xml`.

Reply:
Hmmm, I think I missed something somewhere. Pretty sure it was required for some reason, will look into it.